### PR TITLE
Update mbed_app.json

### PR DESCRIPTION
**Pull request by karsev**
The root level .json updated

### CODE REVIEW

**Comment by 0xc0170** [comment]
Thanks, hopefully will fix the error we are seeing !

File: mbed_app.json [hunk 1]
```diff
@@ -17,13 +19,22 @@
             "mbed-mesh-api.6lowpan-nd-channel-page": 0,
             "mbed-mesh-api.6lowpan-nd-channel": 12,
             "mbed-mesh-api.6lowpan-nd-channel-mask": "(1<<12)",
-            "mbed-mesh-api.thread-config-panid": "0x0700",
-            "mbed-mesh-api.thread-master-key": "{0x10, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff}",
-            "mbed-mesh-api.thread-config-channel": 22,
```
Comment (SeppoTakalo) [inline mbed_app.json]: Why did you remove these lines?
This commit has nothing related to these.
